aardwolf-templates/templates/widgets/icon.rs.html

```diff
@@ -0,0 +1,37 @@
+/// Icon Shortcut Defintions
+/// The list below reduces the code for setting icons in the templates.
+///
+///  Example Usage:
+///  `@icon:"github"`
+///
+/// The example above will be compiled as:
+/// `<span class="fa fa-github"></span>`
+/// 
+/// It is also possible to use the five other sizes provided by FontAwesome
+/// ```
```

Comment:
this is an html file, so rust comments don't make a ton of sense here. look at the [ructe documentation](https://docs.rs/ructe) to see how commenting works